app/abilities/repository.js

```diff
@@ -46,18 +46,6 @@ export default Ability.extend({
         return false;
     }
   }),
-  canToken: computed('currentUser.role_id', 'model.provider.id', function() {
```

Comment:
Please keep this code.

Reply:
I think some of code we're change during the merge. I’ll correct that